docs/Backend_documentation/department_doumentation.md

```diff
@@ -0,0 +1,45 @@
+Department
+----------
+
+Current code:
+@main_blueprint.route("/department/<string:department>")
+def department(department: str):
+    return render_template("department.html")
+
+Possibly:
+What school professor is under:
+- ITWS
+- Lally 
+- School of Architecture
+- School of Engineering
+- HASS
+- School of Science
+
+Professors could post to whole school to widen the range of students that can find the project 
+
+Specific departments/ majors within a school 
+- Computer Science
+- Ect...
+
+Profesors could post to one or maybe more than one major to find specific students within a major
+
+Students would be able to look under schools and under majors
+
+Department:
+    Variables:
+        School:
+            Department:
+
+|Department
+----------
+|ITWS
+|Lally 
+|School of Architecture
+|School of Engineering
+|HASS
+|School of Science 
+
+Therorticly what needs to be done to actually implement this
+- Query the data
+- to get past projects for each profesors
```

Comment:
I think 2 bullet points here would be good. 1 query for current projects in the school/department, 2 query for some past projects, we won't show all of them that could be a separate page or done through ajax (loading on the frontend) but that's an optional thing we can do if we have time later.